Update numerals.mdx (#77)
medzuslovjansky/medzuslovjansky.github.io: docs/learn/grammar/numerals.mdx

The -teens (11-19) are formed by adding `-nadset` (pronounced `-nacet`) to the numbers 1-9:
11\. :isv[jedinnadset], 12. :isv[dvanadset], 13. :isv[trinadset], 14. :isv[četyrinadset], 15. :isv[petnadset], 16. :isv[šestnadset], 17. :isv[sedmnadset], 18. :isv[osmnadset], 19. :isv[devetnadset].

The „-ties” (20-90) are formed by adding `-deset` to the numbers 2-9:
20\. :isv[dvadeset], 30. :isv[trideset], 40. :isv[četyrideset], 50. :isv[petdeset], 60. :isv[šestdeset], 70. :isv[sedmdeset], 80. :isv[osmdeset], 90. :isv[devetdeset].

The hundreds (100-900) are formed by adding `-sto` to the numbers 2-9:
100\. :isv[sto], 200. :isv[dvasto], 300. :isv[tristo], 400. :isv[četyristo], 500. :isv[petsto], 600. :isv[šeststo], 700. :isv[sedmsto], 800. :isv[osmsto], 900. :isv[devetsto]

Alternatively, the hundreds can also be formed by inflecting the word :isv[sto], resulting in the following set:
100\. :isv[sto], 200. :isv[dvěstě], 300. :isv[trista], 400. :isv[četyrista], 500. :isv[petsot], 600. :isv[šestsot], 700. :isv[sedmsot], 800. :isv[osmsot], 900. :isv[devetsot].

> The words for „thousand”, „million” and „milliard” are: :isv[tyseč] (1000), :isv[milion] (106) and :isv[miliard] (109). Just like in the case of the hundreds, these words may or may not be inflected as nouns. Although it is not customary in the Slavic languages to do this, in cases like :isv[pet-tyseć] „5000” a hyphen may be added for clarity.

Reviewer: ```suggestion
The words for „thousand”, „million” and „milliard” are: :isv[tyseč] (1000), :isv[milion] (10⁶) and :isv[miliard] (10⁹). Just like in the case of the hundreds, these words may or may not be inflected as nouns. Although it is not customary in the Slavic languages to do this, in cases like :isv[pet-tyseć] „5000” a hyphen may be added for clarity.
```

### Declension of cardinal numbers \{#declension-of-cardinal-numbers}

The easiest solution for the declension of cardinal numbers is simply not to decline them at all. :isv[Dom s tri etažami] „A house with three floors” is perfectly understandable, even though it may sound a little strange to native speakers. However, a better effect can be achieved by using the following declension patterns:

#### 1 \{#1}

> Except for the masculine nominative singular, the word :isv[jedin] „one” is declined like an adjective **\*:isv[jedny]**: :abbr[m.] **:isv[jedin], :isv[jednogo]**, :abbr[f.] **:isv[jedna], :isv[jednoj]**, :abbr[n.] :isv[jedno], etc. In some cases, it can also be used in the plural: :isv[Jedni ljudi prědpočitajut lěto, drugi zimu] „Some people prefer Summer, others Winter”. This is also the case for pluralia tantum: :isv[jedne dveri] „one door”.

Reviewer: ```suggestion
Except for the masculine nominative singular, the word :isv[jedin] „one” is declined like an adjective \*:isv[jedny]: :abbr[m.] :isv[jedin], :isv[jednogo], :abbr[f.] :isv[jedna], :isv[jednoj], :abbr[n.] :isv[jedno], etc. In some cases, it can also be used in the plural: :isv[Jedni ljudi prědpočitajut lěto, drugi zimu] „Some people prefer Summer, others Winter”. This is also the case for pluralia tantum: :isv[jedne dveri] „one door”.
```

<Numbers24 />

The neuter of "2" is :isv[dva] in Russian, Belarussian, Ukrainian, Polish, Cashubian and Serbocroat, :isv[dvě] in Old Church Slavonic, Czech, Slovak, Upper and Lower Sorbian, Slovene, Macedonian, Bulgarian and Rusyn. In Interslavic, both options are equally valid.

Also declined like :isv[dva] are the words :isv[oba] and :isv[obydva] „both”, „the two of”.

#### 5-99 \{#5-99}

The numbers :isv[pet] and up are inflected like nouns of the pattern :isv[kost]. The subject always remains in the genitive plural. Examples:

<Numbers599 />

#### 0, 100, 1000, 106, 109 \{#0-100-1000-106-109}

The remaining numbers are declined as nouns, too: :isv[nula] „zero” as a feminine noun (like :isv[žena]), :isv[sto] „hundred” as a neuter noun (like :isv[slovo]), :isv[tyseč] „thousand” as a masculine or a feminine noun (:abbr[gen.] :isv[tyseča] or :isv[tyseči]), :isv[milion] „million” and :isv[miliard] „milliard” (or „billion”) as masculine nouns.

### Indefinite numbers \{#indefinite-numbers}

> Apart from the quantitative numbers listed above, there is also a category of so-called _:notr[indefinite cardinal numbers]_, consisting of words like :isv[mnogo] „much, many”, :isv[malo] „little, few”, :isv[velje] „a lot”, :isv[veče] „more”, :isv[menje] „less”, :isv[koliko] „how many”, :isv[několiko] „a few, some”, :isv[toliko] „this/that many”, :isv[para] „a few”.

Reviewer: ```suggestion
Apart from the quantitative numbers listed above, there is also a category of so-called _indefinite cardinal numbers_, consisting of words like :isv[mnogo] „much, many”, :isv[malo] „little, few”, :isv[velje] „a lot”, :isv[veče] „more”, :isv[menje] „less”, :isv[koliko] „how many”, :isv[několiko] „a few, some”, :isv[toliko] „this/that many”, :isv[para] „a few”.
```

They are inflected as feminine nouns. Thus, „7/38” is represented as :isv[sedm trideseti-osmin] (the hyphen is meant to enhance clarity).

## Collective numbers \{#collective-numbers}

The cardinal and ordinal numbers are by far the most significant. However, the Slavic languages have other categories as well. Most of them are rarely used nowadays, or used only under particular circumstances, but they should be mentioned anyway for reasons of completeness.

The collective numbers refer to members a closed group, and can be translated as „a group of”, „-some”. The forms are:
:isv[dvoje] „twosome, pair”, :isv[troje] „threesome, a group of three”, :isv[četvero], :isv[petero], :isv[šestero], :isv[sedmero], :isv[osmero], :isv[devetero], :isv[desetero]. This group also includes :isv[oboje] „both”.

It can be used instead of a cardinal number in the following cases:

> - when the group consists of both men and women: :isv[dva studenti] „two students (:abbr[m.])”, :isv[dvě studentky] „two students (:abbr[f.])”, :isv[dvoje studentov] „two students (:abbr[m. + f.])”

Reviewer: ```suggestion
- when the group consists of both men and women: :isv[dva studenti] „two students (:abbr[m.])”, :isv[dvě studentky] „two students (:abbr[f.])”, :isv[dvoje studentov] „two students (:abbr[m.] + :abbr[f.])”
```

- with gender-neutral words like :isv[ljudi] and :isv[děti]: :isv[osmero ljudij] „eight people”, :isv[četvero dětij] „four children”
- with neuter words denoting young animals (:abbr[sg.] `-e`, :abbr[pl.] `-eta`): :isv[troje telet] „three calves”
- for objects that naturally come in pairs, for example: :isv[dvoje rųkavic] „two gloves”

> - with words that exist only in the plural (_:notr[pluralia tantum]_): :isv[dvoje dverij] „two doors”.

Reviewer: ```suggestion
- with words that exist only in the plural (_pluralia tantum_): :isv[dvoje dverij] „two doors”.
```

As can be seen from these examples, the noun following a collective numeral is always in the genitive plural, and if this noun is the subject of a sentence, the corresponding verb is in the 3rd person singular (neuter).

> Collective numbers can also be used without a noun, in cases like: :isv[my oboje] „both of us (:abbr[m. + f.])”, :isv[zabava v troje] „a party for three”.

Reviewer: ```suggestion
Collective numbers can also be used without a noun, in cases like: :isv[my oboje] „both of us (:abbr[m.] + :abbr[f.])”, :isv[zabava v troje] „a party for three”.
```
